spec/features/idv/doc_auth/document_capture_spec.rb

```
@@ -196,25 +205,28 @@
   end
 
   context 'with a valid passport', allow_browser_log: true do
-    let(:fake_dos_api_endpoint) { 'http://fake_dos_api_endpoint/' }
+    let(:passports_enabled) { true }
 
     before do
-      stub_request(:post, fake_dos_api_endpoint)
-        .to_return(status: 200, body: '{"response" : "YES"}', headers: {})
+      allow(IdentityConfig.store).to receive(:doc_auth_passports_enabled).and_return(true)
+      allow(IdentityConfig.store).to receive(:doc_auth_passports_percent).and_return(100)
+      allow(IdentityConfig.store).to receive(:doc_auth_vendor_default).and_return('mock')
```

specs default to mock

```suggestion
```